projects/plugins/boost/docs/Cache_API.md

```diff
@@ -51,3 +51,38 @@ Usage:
 ```php
 do_action( 'jetpack_boost_clear_page_cache_post', 15 );
 ```
+
+## Filters
+
+### Modify the cache bypass filters
+
+Filter hook: `boost_cache_bypass_patterns`
+
+Parameter: an array of regex patterns that define URLs that bypass caching
+
+Usage:
+```php
+apply_filters( 'boost_cache_bypass_patterns', array( '/shop', '/checkout', '/200?/(.*)' ) );
```

Comment:
The names of the filters changed when we merged https://github.com/Automattic/jetpack/pull/36209.

Reply:
Done. I had updated last week, but I forgot to ping you.